src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -57,7 +57,98 @@ The module connects to the PS3 controller using bluetooth (or USB) and reacts ba
 | SQUARE                    | Toggles locking of the head.               |
 | R2                        | Right kick.                                |
 
-To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and run the binary. Plug the PS3 controller into the robot using a USB cable. When the controller is connected to the robot, it will be paired, and you can remove the cable.
+To use this module, build the [PS3Walk role](https://github.com/NUbots/NUbots/blob/main/roles/ps3walk.role) and install it on the robot. Ensure the PS3 controller is paired and connected to the right robot. Then run the `./ps3walk` on the robot to begin controlling it.
```

Comment:
Wouldn't this be `./b run ps3walk`?

Reply:
No, not on the real robot. That's only in Docker.